pom.xml

```
@@ -21,6 +21,7 @@
         <maven.compiler.target>23</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.shortVersionString>2.1.0</project.shortVersionString> <!-- for Windows MSI versioning -->
+        <lombok.version>1.18.40</lombok.version>
```

Please consider using the latest version of lombok: `1.18.42`

```suggestion
        <lombok.version>1.18.42</lombok.version>
```